app/views/user_sessions/new.html.erb

```diff
@@ -1,47 +1,66 @@
 <div class="container">
+<%= form_for :user_session, :as => :user_session, :url => "/user_sessions?return_to=" + request.fullpath, :html => {:class => "form col-md-6 offset-3"} do |f| %>
 
-<%= form_for :user_session, :as => :user_session, :url => { :action => "create" }, :html => {:class => "form col-md-6 offset-3"} do |f| %>
-
-  <h2><%= raw t('user_sessions.new.log_in_or_sign_up', :url1 => "/signup") %></h2>
+  <h2 id="toSignup"><%= raw t('user_sessions.new.log_in_or_sign_up', :url1 => "#") %></h2>
   <% if f.error_messages != "" %><div class="alert alert-danger"><%= f.error_messages %></div><% end %>
-  <br style="clear:both;"/>
-     <div class='col-lg-10 col-md-10 col-sm-10' style="display:flex; justify-content: center;">
-          <%= render :partial => "layouts/social_icons" %>
-     </div>
-
-  
   <br style="clear:both;"/>
 
-  <div class="form-group">
-
-    <label for="username"><%= t('user_sessions.new.username') %></label>
-    <%= f.text_field :username, { tabindex: 1, class: 'form-control', id: 'username-signup' } %>
-
+  <div class='col-lg-11 col-md-11 col-sm-11' style="display:flex; justify-content: center;">
```

Comment:
Are these changes related to your issue?

Reply:
I've actually renamed the partial _new.html.erb to file new.html.erb and deleted partial _new.html.erb , and changed the header file , 
The thing is that it should have shown the whole file (full green) to change however many things were common in both file and partial that's why its showing like this

Reply:
Can you please adjust it a little bit so that it will easily show up your changes?
It is hard to deduce what you did. I think you will be able to do it in 2-3 commits.
Thanks

Reply:
Sure !

Reply:
@SidharthBansal  should I revert the commits and do them again one by one ?

Reply:
As you wish. Just make it a little more readable.